### PR TITLE
Adds the Floating Resort ruin to Beaches!

### PR DESCRIPTION
**Pull request by Zytolg**
Hello Shiptest new-Friendos! In case you don't know me, I'm Zytolg from over at /tg/. I've done a ton of mapping for various SS13 related projects, and I've finally bitten the shiptest hook. Please treat me well and thanks for having me!

## About The Pull Request

Adds a new beach ruin to shiptest.

## Why It's Good For The Game

More ruins = more variety. Ya'll asked me to add ruins to Beach/Rock/Scrap ruins so I'll be doing that this month :3

Forgot the image here it is
La_vida_Loca_Shiptest_Map

## Changelog

:cl:
add: Added the Floating Resort to Beaches. Enjoy your moment in paradise!
/:cl:

### CODE REVIEW

**Comment by Valorium-RB** [comment]
Would it be possible to place a screenshot of the ruin in your PR? Thanks!

**Comment by MarkSuckerberg** [comment]
Forgive the harddel errors, they're not you're fault and I'll try to rerun it. Once #2364 gets merged it shouldn't happen anymore.

File: code/game/area/areas/ruins/beachplanet.dm [hunk 1]
```diff
@@ -69,3 +69,16 @@
...
 	name = "Pirate Cavern"
 	icon_state = "purple"
+
+//beach_float_resort --> keeping resort open for a land based ruin
+
+/area/ruin/beach/float_resort
+	name = "Beach Resort"
+	icon_state = "yellow"
+	mood_bonus = 10
+	mood_message = "<span class='nicegreen''>Incredible! A hidden resort all to myself and my crew! </span>\n"
```
Comment (goober3) [inline code/game/area/areas/ruins/beachplanet.dm]: this is a neat idea, but - please do not do this! moodlets tend to detract from RP.

**Review by goober3 — DISMISSED** [review]
please remember to give this ruin a datum (code/datums/ruins/beachplanet.dm) & update the map catalogue! it will not spawn in-game, otherwise

also, expand the map & blend the terrain / water with some turf passthroughs, or the border between the planet and the ruin itself will look strange and unnatural! 

image
remove the siding decals around the grass here, the grass creates sidings itself, which will overlap other sidings.

please add more (minor) loot & greebles to the floating resort area -- consider, a fishing area, or a dock / ladder, as well as more appliances in the villas - microwaves, lamps, refrigerators, etcetera. little tidbits of loot to satisfy scrappers.

**Review by spockye — APPROVED** [review]
seems good to me!
thanks for making ruins <3

**Pull request merged**